index.html

```diff
@@ -207,6 +207,23 @@ <h5 class="text2">Skills</h5>
     </ul>
 
     <div class="seperator"></div>
+    <div class="contact-form">
+        <span>Contact me</span>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+        <form action="https://formspree.io/f/xzbywprd" method="POST">
+        <input type="text" name="name" id="name" placeholder="Yeremias">
+        <input type="text" name="name" id="email" placeholder="natajayanj14@gmail.com">
+        <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here"></textarea>
+        <input type="button" class="contact-form-btn" value="Get It Touch">
```

Comment:
- [ ] The type attribute for the submit button should be `submit`. This is the best way to link it to the form.